app/domain/search.rb

```diff
@@ -38,13 +44,28 @@ def page=(value)
 
   def sort=(identifier)
     query.sort = identifier
+    raise_if_unrecognised_sort
```

Comment:
It might be more consistent to move this to `Query` with the other raises. What do you think?